lib/rucio/core/did_meta_plugins/did_meta_plugin_interface.py

```diff
@@ -35,20 +33,32 @@ def __init__(self):
         """
         Initializes the plugin
         """
-        pass
+        self._plugin_name = None
+
+    @property
+    def name(self) -> str:
+        """
+        The getter method for the plugin's name.
+
+        :returns: The standardized (uppercase) name of this plugin.
```

Comment:
This is a bit confusing because the returned name is actually lowercase, though this says uppercase (I guess the uppercase here refers to the original case of the name)